doc/markdown/Group_LASSO.md

```diff
@@ -39,6 +36,14 @@ B = gg1$beta
 L = gg1$lambda
 Lr = gglasso(x=Xr,y=Y,group=group1,loss="ls")$lambda
 DF = gg1$df
+''')
+B = rpy2.r('B')
```

Comment:
Sounds good -- applied this here

https://github.com/jonathan-taylor/regreg/commit/811f23bb76952c2219f75f4def6a1b42b46b2800